app/src/pages/playground/__tests__/playgroundUtils.test.ts

```diff
@@ -78,6 +78,8 @@ describe("transformSpanAttributesToPlaygroundInstance", () => {
     expect(transformSpanAttributesToPlaygroundInstance(span)).toStrictEqual({
       playgroundInstance: {
         ...expectedPlaygroundInstanceWithIO,
+        // do we still want to set spanId if we can't parse the attributes?
+        spanId: null,
```

Comment:
Not sure what the harm would be, if any. Tempted to do this

Reply:
i think if we can't parse the attributes we can still set it, i think the philosophy has been try to parse as much as you can but don't let any failures prevent you from getting other information. That is keep each parsing isolated from the other

Reply:
Done